reduce/src/adapter.ts

```diff
@@ -1,7 +1,7 @@
 import objectPath from 'object-path'
 import { Requester, Validator } from '@chainlink/external-adapter'
 import { Execute } from '@chainlink/types'
-import BN from 'bignumber.js'
+import { Decimal } from 'decimal.js'
```

Comment:
Something I've noticed is that Decimal.js likes to return scientific notation. I would assume everything we use would be able to understand that, but is that something we need to consider more?

Reply:
Two situations that I can think of are:

- When converting from `Decimal` to `BigNumber`, as `BigNumber` does not support scientific notation
- When printing decimal numbers to the output, we don't want the scientific notation there

Reply:
This is now safeguarded against in the `reduce` adapter.